### ISSUE TITLE
Twig error in Craft 3.0.23

### ISSUE THREAD

**Issue by brandonkelly** [engram-design/SuperTable#194]
Craft’s `_components/fieldtypes/Matrix/settings.html` template has changed for the upcoming v3.0.23 release (craftcms/cms@eeac5b670583167704f86196f1b9323aa19aeb5b), which causes a Twig error when creating or editing any custom fields:

> Variable "blockTypes" does not exist.

**Comment by itsmattsoria** [comment]
Just ran into this error myself — any word on when this fix will be added to a Super Table update @engram-design?

**Comment by brandonkelly** [comment]
@poopsplat you should be able to get the fix ahead of time by changing your `verbb/super-table` version constraint in composer.json to `"dev-craft-3 as 2.0.8"`, and then run `composer update`.

```json
"require": {
    "...": "...",
    "verbb/super-table": "dev-craft-3 as 2.0.8"
}
```

**Comment by ademers** [comment]
@brandonkelly Thanks for this! I'm not clear on how to implement your solution. Would you please show us what this looks like in the composer.json file?

**Comment by brandonkelly** [comment]
@ademers just updated my previous comment with an example :)

**Comment by itsmattsoria** [comment]
@brandonkelly thank you 🙌

**Comment by BramBeekman17** [comment]
@brandonkelly Thanks man!

**Comment by janhendrickx** [comment]
@brandonkelly Thanks for the fix!

**Comment by daniellelecomte** [comment]
@brandonkelly Thanks so much!

**Comment by XhmikosR** [comment]
@engram-design: any reason there isn't a new patch version out? This is quite an important fix.

**Comment by controlnocontrol** [comment]
I've done this but composer says there's nothing to update.
`Nothing to install or update`
Help?

**Comment by brandonkelly** [comment]
@ohbiko 

- Make sure you actually saved `composer.json`.
- Try running `composer clear-cache`.
- Make sure you're running `composer update`, **not** `composer install`.

**Comment by engram-design** [comment]
Sorry for the delay all, this should be fixed in [2.0.9](https://github.com/verbb/super-table/releases/tag/2.0.9).
